README.md

```diff
@@ -158,6 +159,33 @@ $ java -jar xpanse-runtime-1.0.0-SNAPSHOT.jar --spring.profiles.active=zitadel \
 --authorization-swagger-ui-client-id=221664356859969539@eclipse-xpanse
 ```
 
+##### Execute authenticated APIs
+
+When the application activated authorization with 'zitadel', execute authenticated APIs must add a header
```

Comment:
When the application has activated authorization with 'zitadel' profile, all protected APIs must have an 'Authorization header in the `Bearer ${access token}` format in the http request. Here are two ways to get
`access_token` for executing authenticated APIs:

Reply:
fixed.